source/Octodiff.Tests/Core/BinaryDeltaReaderFixture.cs

```diff
@@ -2,6 +2,7 @@
 using System.IO;
 using System.Linq;
 using NUnit.Framework;
+using NUnit.Framework.Legacy;
```

Comment:
I'm not familiar enough with NUnit, nor it's upgrade path, to know what has/hasn't been turned into `ClassicAssert`s.

That said, this is the only change in this file. Is that intentional?

Reply:
Yeah, it is.
This file uses `CollectionAssert` in quite a few places, and that type moved into the Legacy namespace. We need this extra using statement or it won't compile.